stdlib/subprocess.pyi

```diff
@@ -804,7 +804,556 @@ class Popen(Generic[AnyStr]):
+            universal_newlines: bool = ...,
+            startupinfo: Any | None = ...,
+            creationflags: int = ...,
+            restore_signals: bool = ...,
+            start_new_session: bool = ...,
+            pass_fds: Any = ...,
+            *,
+            text: Literal[True],
+            encoding: str | None = ...,
+            errors: str | None = ...,
+            user: str | int | None = ...,
+            group: str | int | None = ...,
+            extra_groups: Iterable[str | int] | None = ...,
+            umask: int = ...,
+            pipesize: int = ...,
+        ) -> Popen[str]: ...
+        @overload
+        def __new__(
+            cls,
+            args: _CMD,
+            bufsize: int = ...,
+            executable: StrOrBytesPath | None = ...,
+            stdin: _FILE | None = ...,
+            stdout: _FILE | None = ...,
+            stderr: _FILE | None = ...,
+            preexec_fn: Callable[[], Any] | None = ...,
+            close_fds: bool = ...,
+            shell: bool = ...,
+            cwd: StrOrBytesPath | None = ...,
+            env: _ENV | None = ...,
+            universal_newlines: Literal[False] = ...,
+            startupinfo: Any | None = ...,
+            creationflags: int = ...,
+            restore_signals: bool = ...,
+            start_new_session: bool = ...,
+            pass_fds: Any = ...,
+            *,
+            text: Literal[None, False] = ...,
+            encoding: None = ...,
+            errors: None = ...,
+            user: str | int | None = ...,
+            group: str | int | None = ...,
+            extra_groups: Iterable[str | int] | None = ...,
+            umask: int = ...,
+            pipesize: int = ...,
+        ) -> Popen[bytes]: ...
+        @overload
+        def __new__(
+            cls,
+            args: _CMD,
+            bufsize: int = ...,
+            executable: StrOrBytesPath | None = ...,
+            stdin: _FILE | None = ...,
+            stdout: _FILE | None = ...,
+            stderr: _FILE | None = ...,
+            preexec_fn: Callable[[], Any] | None = ...,
+            close_fds: bool = ...,
+            shell: bool = ...,
+            cwd: StrOrBytesPath | None = ...,
+            env: _ENV | None = ...,
+            universal_newlines: bool = ...,
+            startupinfo: Any | None = ...,
+            creationflags: int = ...,
+            restore_signals: bool = ...,
+            start_new_session: bool = ...,
+            pass_fds: Any = ...,
+            *,
+            text: bool | None = ...,
+            encoding: str | None = ...,
+            errors: str | None = ...,
+            user: str | int | None = ...,
+            group: str | int | None = ...,
+            extra_groups: Iterable[str | int] | None = ...,
+            umask: int = ...,
+            pipesize: int = ...,
+        ) -> Popen[Any]: ...
+    elif sys.version_info >= (3, 9):
+        # user, group, extra_groups, umask were added in 3.9
```

Comment:
These shouldn't be available on windows, but we probably shouldn't change the stub to reflect that. It's already difficult enough to read. The same goes for `process_group`.